### ISSUE TITLE
Mehrere Kategorien auswählen?

### ISSUE THREAD

**Issue by AndiWebred** [RRZE-Webteam/rrze-calendar#167]
Je nachdem wie Kategorien verwendet werden, kann es sein, dass eine Veranstaltung mehrere Kategorien bräuchte. Momentan kann man nur eine einstellen.

Nicht super dringend, weil man kann ja zur Not auch mit Schlagworten arbeiten.

**Comment by cassandre** [comment]
Das Problem ist dann, in welcher Farbe wird der Termin angezeigt, wenn mehrere Kategorien mit unterschiedlichen Farben gewählt werden?

**Comment by AndiWebred** [comment]
Ja, stimmt, daran hab ich nicht gedacht. Bei Posts können wir mittlerweile eine "Pimäre Kategorie" anklicken. Das wäre dann die primäre Farbe. Aber eventuell führt das dann dazu, dass ich in einem roten Kalender einen blauen Termin habe. Das muss mir halt bewusst sein, wenn ich mehrere Kategorien wähle.

**Comment by rvdforst** [comment]
Vielleicht gibt es eine Lösung:
1. Nur Hauptkategorien kann eine Farbe zugewiesen werden
2. Unterkategorien erben die Farbe der übergeordneten Kategorie
Wenn wir uns einig sind, ist dies relativ einfach zu implementieren (tatsächlich vertraue ich dem aktuellen Code, der ein Kontrollkästchen in ein Radio-Button umwandelt, nicht).
